src/parser/parser.yy

```diff
@@ -460,6 +471,37 @@ adt_branch
     }
   ;
 
+/**
+ * Lattice Declarations
+ */
+
+lattice_decl
+  : LATTICE IDENT[name] LT GT LBRACE lattice_operator_list RBRACE
```

Comment:
I got inspiration from Flix. Because lattice arguments in relation are also suffixed with `<>`, I also used that same name in the `.lattice`declaration. This can be discussed.